### PR TITLE
{Keyvault} Fix#21487: `az keyvault key create`: Fix keyvault createkey links issue

### LINKED ISSUE #21487: Links to cert info not found.
[https://docs.microsoft.com/rest/api/keyvault/createkey/createkey#jsonwebkeycurvename](https://docs.microsoft.com/en-us/rest/api/keyvault/createkey/createkey#jsonwebkeycurvename).

And

[https://docs.microsoft.com/rest/api/keyvault/createkey/createkey#jsonwebkeytype](https://docs.microsoft.com/en-us/rest/api/keyvault/createkey/createkey#jsonwebkeytype).

retuns 404. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fe0f363b-7f21-0ea9-3e27-da7866c94102
* Version Independent ID: 6cc413e4-4f7d-9baa-a365-c9275fc6a7ad
* Content: [az keyvault key](https://docs.microsoft.com/en-us/cli/azure/keyvault/key?view=azure-cli-latest#az-keyvault-key-create)
* Content Source: [latest/docs-ref-autogen/keyvault/key.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/latest/docs-ref-autogen/keyvault/key.yml)
* Service: **key-vault**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

### PR DESCRIPTION
**Pull request by jonie001**
**Description**

Fix #21487 

**Testing Guide**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Keyvault

File: src/azure-cli/azure/cli/command_modules/keyvault/_params.py [hunk 1]
```diff
@@ -453,9 +453,9 @@ class CLISecurityDomainOperation(str, Enum):
 
     with self.argument_context('keyvault key create') as c:
         c.argument('kty', arg_type=get_enum_type(JsonWebKeyType), validator=validate_key_type,
-                   help='The type of key to create. For valid values, see: https://docs.microsoft.com/rest/api/keyvault/createkey/createkey#jsonwebkeytype')
+                   help='The type of key to create. For valid values, see: https://docs.microsoft.com/en-us/rest/api/keyvault/keys/create-key/create-key#jsonwebkeytype')
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/keyvault/_params.py:456]:
```suggestion
                   help='The type of key to create. For valid values, see: https://docs.microsoft.com/rest/api/keyvault/keys/create-key/create-key#jsonwebkeytype')
```

File: src/azure-cli/azure/cli/command_modules/keyvault/_params.py [hunk 2]
```diff
@@ -453,9 +453,9 @@ class CLISecurityDomainOperation(str, Enum):
...
         c.argument('curve', arg_type=get_enum_type(KeyCurveName),
-                   help='Elliptic curve name. For valid values, see: https://docs.microsoft.com/rest/api/keyvault/createkey/createkey#jsonwebkeycurvename')
+                   help='Elliptic curve name. For valid values, see: https://docs.microsoft.com/en-us/rest/api/keyvault/keys/create-key/create-key#jsonwebkeycurvename')
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/keyvault/_params.py:458]:
```suggestion
                   help='Elliptic curve name. For valid values, see: https://docs.microsoft.com/rest/api/keyvault/keys/create-key/create-key#jsonwebkeycurvename')
```

**Review by jiasli — APPROVED** [review]
Approve as long as `en-us/` is removed.

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**